CHANGES.md

```diff
@@ -15,6 +15,19 @@
 
   * Compatibility with GHC 9.0
 
+  * Improve handling of XDG directories.
+
+      1. If all three of xmonad's environment variables (`XMONAD_DATA_DIR,`
+         `XMONAD_CONFIG_DIR`, and `XMONAD_CACHE_DIR`) are set, use them.
+      2. If there is a build script `build` or configuration `xmonad.hs` in
```

Comment:
What does build script `build` mean ?

Reply:
I'm referring to something like `~/.xmonad/build` or `~/.config/xmonad/build`; there are [examples](https://github.com/xmonad/xmonad-testing/tree/master/build-scripts) for this in the `xmonad-testing` repository. 

I'll try to improve the docs a bit there